eq-author/src/components/Forms/WrappingInput/index.js

```diff
@@ -38,11 +41,58 @@ class WrappingInput extends React.Component {
     bold: false,
   };
 
+  state = { show: false, text: "", event: {} };
```

Comment:
From conversation on a call with @Paul-Joel:

The `show` value can be renamed to describe the content being shown, e.g. `showModal`

Reply:
Renamed